core/src/main/java/cz/o2/proxima/repository/AttributeFamilyDescriptor.java

```diff
@@ -135,6 +89,45 @@ public static Builder newBuilder() {
     this.access = Objects.requireNonNull(access);
     this.filter = filter;
     this.source = source;
+
+    this.replicationConsumerNameFactory =
+        constructConsumerNameFactory(
+            cfg,
+            CFG_REPLICATION_CONSUMER_NAME_GENERATOR,
+            "consumer-",
+            DefaultReplicationConsumerNameFactory.CFG_REPLICATION_CONSUMER_NAME_PREFIX,
+            DefaultReplicationConsumerNameFactory.CFG_REPLICATION_CONSUMER_NAME_SUFFIX);
+    this.transformationConsumerNameFactory =
+        constructConsumerNameFactory(
+            cfg,
+            CFG_TRANSFORMATION_CONSUMER_NAME_GENERATOR,
+            "transformer-",
+            DefaultReplicationConsumerNameFactory.CFG_TRANSFORMER_CONSUMER_NAME_PREFIX,
+            DefaultReplicationConsumerNameFactory.CFG_TRANSFORMER_CONSUMER_NAME_SUFFIX);
+  }
+
+  private ReplicationConsumerNameFactory constructConsumerNameFactory(
```

Comment:
This should return `ConsumerNameFactory`, because it is used for `transformationConsumerNameFactory` and `replicationConsumerNameFactory`